paths/graphql/04-query-info.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Grab a Query's Response
+permalink: /graphql/query-info
+next-page: /graphql/another-query
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  1. Using the response from the query in the last step, fill in this template.
+
+          My most recent PRs are: <br>
```

Comment:
I love this idea, and the end result is really cool. However, do you think that this might be just as effective with 1 or 2 PR's?